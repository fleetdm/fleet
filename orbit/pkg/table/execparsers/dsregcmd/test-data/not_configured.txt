
+----------------------------------------------------------------------+
| Device State                                                         |
+----------------------------------------------------------------------+

             AzureAdJoined : NO                                         
          EnterpriseJoined : NO                                         
              DomainJoined : NO                                         
               Device Name : lenovo-thinkpad                            
                                                                        
+----------------------------------------------------------------------+
| User State                                                           |
+----------------------------------------------------------------------+
                                                                        
                    NgcSet : NO                                         
           WorkplaceJoined : NO                                         
             WamDefaultSet : ERROR                                      
                                                                        
+----------------------------------------------------------------------+
| SSO State                                                            |
+----------------------------------------------------------------------+
                                                                        
                AzureAdPrt : NO                                         
       AzureAdPrtAuthority : NO                                         
             EnterprisePrt : NO                                         
    EnterprisePrtAuthority : NO                                         
                                                                        
+----------------------------------------------------------------------+
| IE Proxy Config for Current User                                     |
+----------------------------------------------------------------------+

      Auto Detect Settings : YES
    Auto-Configuration URL :
         Proxy Server List :
         Proxy Bypass List :

+----------------------------------------------------------------------+
| WinHttp Default Proxy Config                                         |
+----------------------------------------------------------------------+

               Access Type : DIRECT

+----------------------------------------------------------------------+
| Ngc Prerequisite Check                                               |
+----------------------------------------------------------------------+

            IsDeviceJoined : NO
             IsUserAzureAD : NO
             PolicyEnabled : NO
          PostLogonEnabled : YES
            DeviceEligible : YES
        SessionIsNotRemote : YES
            CertEnrollment : none
              PreReqResult : WillNotProvision

For more information, please visit https://www.microsoft.com/aadjerrors

