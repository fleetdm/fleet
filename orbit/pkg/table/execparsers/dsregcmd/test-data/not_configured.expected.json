{
  "Device State": {
    "AzureAdJoined": "NO",
    "Device Name": "lenovo-thinkpad",
    "DomainJoined": "NO",
    "EnterpriseJoined": "NO"
  },
  "IE Proxy Config for Current User": {
    "Auto Detect Settings": "YES",
    "Auto-Configuration URL": "",
    "Proxy Bypass List": "",
    "Proxy Server List": ""
  },
  "Ngc Prerequisite Check": {
    "CertEnrollment": "none",
    "DeviceEligible": "YES",
    "For more information, please visit https": "//www.microsoft.com/aadjerrors",
    "IsDeviceJoined": "NO",
    "IsUserAzureAD": "NO",
    "PolicyEnabled": "NO",
    "PostLogonEnabled": "YES",
    "PreReqResult": "WillNotProvision",
    "SessionIsNotRemote": "YES"
  },
  "SSO State": {
    "AzureAdPrt": "NO",
    "AzureAdPrtAuthority": "NO",
    "EnterprisePrt": "NO",
    "EnterprisePrtAuthority": "NO"
  },
  "User State": {
    "NgcSet": "NO",
    "WamDefaultSet": "ERROR",
    "WorkplaceJoined": "NO"
  },
  "WinHttp Default Proxy Config": {
    "Access Type": "DIRECT"
  }
}
